src/sidebar/style.css

```diff
@@ -170,16 +170,19 @@ li {
 }
 
 
-.container-tab-dragged-over,
-.container-dragged-over .container-header {
+.container-dragged-over .container-header,
+.container-dragged-over .container-tab,
+.tab-dragged-over.container-tab,
+.tab-dragged-over .container-header
+ {
     border-bottom: 1px solid #eee !important;
 }
 
-#pinned-tabs .container-tab-dragged-over {
-    border-right: 1px solid #eee;
```

Comment:
This border-right was used when moving tab from unpinned to a pinned state.  Now there's no indicator where the tab will land.

Reply:
OK